tests/test_save_fields.py

```diff
@@ -93,3 +93,15 @@ def test_correctly_handle_foreignkeys_id_field_in_update_fields():
 
     tmwfk.save(update_fields=['fkey'])
     assert tmwfk.get_dirty_fields(check_relationship=True) == {}
+
+
+@pytest.mark.django_db
+def test_save_deferred_field_with_update_fields():
+    TestModel.objects.create()
+
+    tm = TestModel.objects.defer('boolean').first()
+    tm.boolean = False
+
+    assert tm.get_dirty_fields() == {}
+
+    tm.save(update_fields=['boolean'])
```

Comment:
it's seem good for me. Maybe i can add test in an another PR @romgar ? (like in : https://github.com/mrjmad/django-dirtyfields/commit/6728016afa73d8f7b946a21773b28443e60914e6 )